packages/modals/src/elements/DrawerModal/DrawerModal.spec.tsx

```diff
@@ -83,15 +83,15 @@ describe('DrawerModal', () => {
 
     userEvent.click(getByText('Open Drawer'));
 
-    expect(getByText('title')).toHaveAttribute('id', `${DRAWER_MODAL_ID}--title`);
+    expect(getByText('title')).toHaveAttribute('id', `${DRAWER_MODAL_ID}__title`);
```

Comment:
This is expected since `react-containers` was updated (to use the proper BEM notation). I think we can assume subcomponent IDs are under Garden's control and out-of-scope for end user impact.